workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -165,12 +196,18 @@ private static bool BuildWorkers(string[] workerTypes, BuildEnvironment buildEnv
             }
         }
 
-        private static bool BuildWorkerForEnvironment(string workerType, BuildEnvironment targetEnvironment, ScriptingImplementation? scriptingBackend = null)
+        private static bool BuildWorkerForEnvironmentTargets(
+            string workerType,
+            BuildEnvironment buildEnvironment,
+            IEnumerable<BuildTarget> buildTargets,
+            ScriptingImplementation? scriptingBackend = null)
         {
             var spatialOSBuildConfiguration = BuildConfig.GetInstance();
-            var environmentConfig = spatialOSBuildConfiguration.GetEnvironmentConfigForWorker(workerType, targetEnvironment);
+            var environmentConfig = spatialOSBuildConfiguration.GetEnvironmentConfigForWorker(workerType, buildEnvironment);
 
-            var enabledTargets = environmentConfig?.BuildTargets.Where(t => t.Enabled).ToList();
+            var enabledTargets = buildTargets.Any()
+                ? environmentConfig?.BuildTargets.Where(t => buildTargets.Contains(t.Target)).ToList()
```

Comment:
What about the `Enabled` property?

Reply:
I think if we pass in a set of targets, the expectation is that the target gets built regardless of whether the build config sets it as enabled